.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/DatePeriodParser.cs

```diff
@@ -1,18 +1,17 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Globalization;
-using DateObject = System.DateTime;
-
-using Microsoft.Recognizers.Text.Number.Japanese;
 using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.Number.Japanese;
+using DateObject = System.DateTime;
 
 namespace Microsoft.Recognizers.Text.DateTime.Japanese
 {
     public class DatePeriodParser : IDateTimeParser
     {
-        private const int LastMonthOfYear = 12;
+        public static readonly string ParserName = Constants.SYS_DATETIME_DATEPERIOD; // "DatePeriod";
 
-        public static readonly string ParserName = Constants.SYS_DATETIME_DATEPERIOD; //"DatePeriod";
+        private const int LastMonthOfYear = 12;
 
         private static readonly IDateTimeExtractor SingleDateExtractor = new DateExtractor();

```

Comment:
All static properties should be togther, shouldn't them?

Reply:
The order according to StyleCop rules is First all public variables static or not static then protected finally private in the same way.

Reply:
ACK